src/explorer/kafkaExplorer.ts

```diff
@@ -114,4 +114,18 @@ export class KafkaExplorer implements vscode.Disposable, vscode.TreeDataProvider
             focus: true
         });
     }
+
+    public async copyToClipboard(node: NodeBase | undefined): Promise<void> {
+        if (!node) {
+            //get selected tree item
+            const selection = this.tree?.selection;
```

Comment:
always empty. WTF?

Reply:
There should be second argument, which is array of selected elements in tree
``` 
/**
		 * Whether the tree supports multi-select. When the tree supports multi-select and a command is executed from the tree,
		 * the first argument to the command is the tree item that the command was executed on and the second argument is an
		 * array containing all selected tree items.
		 */
		canSelectMany?: boolean;
```

Reply:
You also need to pass that option when creating tree here https://github.com/jlandersen/vscode-kafka/pull/78/files#diff-5a5b30618f14f5caa7d606bbcf3d67de2ae226c596f4d96dd7f5bb5cb4c2e6e8L37

Reply:
Just verified if `tree.selection' and it works.

Reply:
@dgolovin thanks for figuring out my issue!